frontend/public/components/notification-drawer.tsx

```diff
@@ -38,22 +40,33 @@ import {
 import { getSortedUpdates } from './modals/cluster-update-modal';
 import { usePrevious } from '@console/metal3-plugin/src/hooks';
 
-const emptyState = (toggleExpanded: () => void) => (
+const errorState = (
   <EmptyState variant={EmptyStateVariant.full} className="co-status-card__alerts-msg">
-    <Title headingLevel="h5" size="lg">
-      No critical alerts
-    </Title>
-    <EmptyStateBody>
-      There are currently no critical alerts firing. There may be firing alerts of other severities
-      or silenced critical alerts however.
-    </EmptyStateBody>
-    <EmptyStateSecondaryActions>
-      <Link to="/monitoring/alerts" onClick={toggleExpanded}>
-        View all alerts
-      </Link>
-    </EmptyStateSecondaryActions>
+    <EmptyStateIcon className="co-status-card__alerts-icon" icon={UnknownIcon} />
+    <EmptyStateBody>Alerts could not be loaded.</EmptyStateBody>
```

Comment:
It would be nice to show the actual error.

Reply:
@spadgett speaking with @cshinn  we decided on using the same format as the dashboard. Do you want me to add more detail if available?

Reply:
It will be hard to troubleshoot what's wrong if we don't show any error details.

The dashboard page is maybe a little different because we want to avoid showing the same error again and again on every card if prometheus is down. @cshinn ?

Reply:
I was unaware that we would get any more information other than "not responding" If we have a more useful message to show, I think it makes sense to do so

Reply:
cool, @cshinn do you have an idea for where to display the actual error?